### DISCUSSION TITLE
Delete data in Guilds by Glare

### DISCUSSION THREAD

**Discussion by Naghell** [Zrips/CMI#6149] [Enhancement]
Hi, I have a server that uses CMI and Guilds. The main problem is that my server deletes the data of the inactive users for 6 months, and don't delete the users from their respective Guilds.
I talk to the dev of the Guilds plugin and he told me that it requires integration with their API. Is there a possibility to implement this function?

**Comment by mrfloris** [comment]
Glare's a pretty cool person, how are you purging inactive players with CMI in the first place (so I don't misunderstand you).

**Comment by Naghell** [comment]
I use the option to automatically purge inactive players, in config.
